run.sh

```diff
@@ -46,6 +46,7 @@ gpg --allow-secret-key-import --import tests/files/securedrop.gpg.asc &
 
 # check whether current env is qubes
 is_qubes="$(printenv | grep ^QUBES_)" || true
+is_wayland="$(echo $XDG_SESSION_TYPE | grep ^wayland)" || true
```

Comment:
Is there a reason we're grepping for it instead of a simple equality check?

Reply:
No :upside_down_face:  (well, yes, I wasn't 100% sure if suffixes like `wayland-0` were valid or not, but looks like not). Fixing!

Sorry this has been sitting for 500 years